cpp/src/encoding/gorilla_encoder.h

```diff
@@ -120,6 +120,7 @@ class GorillaEncoder : public Encoder {
     }
 
     int get_one_item_max_size();
+    int get_max_byte_size();
```

Comment:
What is the difference between the two? Their implementations are identical.

Reply:
It's redundant. I'll delete get_one_item_max_size();